lib/components/user/types.ts

```diff
@@ -25,16 +25,23 @@ export type VisionLimitation = typeof visionLimitations[number]
 export interface MobilityProfile {
   isMobilityLimited: boolean
   mobilityDevices: string[]
+  mobilityMode: string
   visionLimitation: VisionLimitation
 }
 
+export interface CompanionInfo {
+  email: string
+  status?: string
+}
+
 /**
  * Type definition for an OTP-middleware (OTP-personas) user.
  */
 export interface User {
   accessibilityRoutingByDefault?: boolean
   // email always exists per Auth0.
   email: string
+  guardians?: CompanionInfo[]
```

Comment:
The field `guardians` has been renamed in the backend to `relatedUsers` at my request.